packages/core/src/Input/Input.tsx

```diff
@@ -107,32 +108,41 @@ export interface HvInputProps<
   /** The error message to show when `status` is "invalid". */
   statusMessage?: string;
   /** @inheritdoc */
-  onChange?: (event: React.ChangeEvent<InputElement>, value: string) => void;
+  onChange?: (
+    event: React.ChangeEvent<HTMLInputElement>,
```

Comment:
We can't do this... `InputElement` is the generic (maybe I could've named it something more obvious like `TInput` or `InputType`), and could be a TextArea or something that extends input.

We just need to do `HvInputProps<HTMLInputElement>` on the other side